server/middleware/auth.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-else-return */
+/* eslint-disable consistent-return */
+/* eslint-disable no-console */
+import jwt from 'jsonwebtoken';
+import config from '../config';
+
+function invalidResponse(response, message) {
```

Comment:
Abstract invalid reaponse to  util file so that you can use it at multiple places